src/auth/token-verifier.ts

```diff
@@ -320,6 +396,34 @@ export class FirebaseTokenVerifier {
       });
   }
 
+  /* eslint-disable */
+  /** @alpha */
+  public _verifyAuthBlockingToken(
+    jwtToken: string,
+    isEmulator: boolean,
+    audience?: string): Promise<DecodedAuthBlockingToken> {
+    if (!validator.isString(jwtToken)) {
+      throw new FirebaseAuthError(
+        AuthClientErrorCode.INVALID_ARGUMENT,
+        `First argument to ${this.tokenInfo.verifyApiName} must be a ${this.tokenInfo.jwtName} string.`,
```

Comment:
The error messages will read as `this.tokenInfo.jwtName === Firebase ID token` and `this.tokenInfo.verifyApiName === verifyIdToken()`. Is that okay? Otherwise we should create a new `FirebaseTokenInfo` for blocking tokens that will also require changes to `createIdTokenVerifier()`

Reply:
So I'll ask for your guidance here. I was originally leaving it as an id token verifier since the blocking tokens are extremely similar to id tokens and we're not going to release this externally. But I'm not sure if that's the right approach if we are going to keep this long term. What do you think?

Reply:
Okay I went ahead and added in the auth blocking token verifier